### PR TITLE
Hello world project

### PR DESCRIPTION
**Pull request by Ibranista**
added html file with a heading, having a class of h1. And created a stylesheet for the header (h1) section.

### CODE REVIEW

File: hello.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+html{
+  background:darkorange;
+}
+h1{
+  color:skyblue;
+}
```
Comment (Abenezer-Tilahun) [inline hello.css:6]: - [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
Capture

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```
Comment (Abenezer-Tilahun) [inline test.md:1]: - [ ] - [ ] As stated in the project requirement A test.md file with the text "This file should be ignored by git" is actually ignored by git and is not present in your GitHub repository. but still i see it on your Repo.
Capture

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.gitignore
```
Comment (Abenezer-Tilahun) [inline .gitignore:1]: - [ ] Please try to remove the .gitingore file .

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Ibranista   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻


✔️ You used correct GitHub Flow.
✔️ Professional README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ]  I recommend that you fix the linters errors in your styles.css. To have the project approved, the linter result should be green. AKA: Linter should report NO errors.
- [ ] Your  Pull Request should have a descriptive title telling you what changes are inside and have a short summary describing introduced changes in more detail.
Capture

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 3 above)
Comment (omarsalem7) [inline .gitignore:1]: - [ ] As stated by the previous reviewer, kindly remove `.gitignore` from the `.gitignore` file.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Ibranista , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
- [ ] Kindly make sure to fix linter errors and make them green by clicking in details and seeing the error and fixing it because it is one of the core requirements.
- [ ] Also consider adding a description for pull request by clicking on 3 dots and then editing the description.
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Ibranista      :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**